aten/src/ATen/core/PythonFallbackKernel.cpp

```diff
@@ -88,6 +90,19 @@ void pythonFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
           return;
         }
       }
+    } else if (ivalue.isSymIntList()) {
+      for (const auto& nv : ivalue.toListRef()) {
+        if (nv.isNone()) {
+          continue;
+        }
+        auto x = nv.toSymInt();
```

Comment:
Maybe you can avoid the refcount bump here, not sure.

Reply:
Updated!